### PR TITLE
add link to vscode to repo header

### PR DESCRIPTION
**Pull request by a1012112796**
example:
log2

may be can add more ide, But I don't know the link format, feel free to add it if you like.

### CODE REVIEW

File: templates/repo/clone_buttons.tmpl [hunk 1]
```diff
@@ -1,3 +1,19 @@
...
+				<strong>{{.i18n.Tr "repo.clone_in_ide"}}</strong>
+			</div>
+			<a class="item" href="vscode://vscode.git/clone?url={{if $.PageIsWiki}}{{$.WikiCloneLink.HTTPS}}{{else}}{{$.CloneLink.HTTPS}}{{end}}">
+				VS Code
+			</a>
+		</div>
+	</div>
+{{end}}
+
 {{if not $.DisableHTTP}}
```
Comment (lunny) [inline templates/repo/clone_buttons.tmpl]: Maybe insert them here so that we could only check `$.DisableHTTP` once.

**Comment by lunny** [comment]
It's not aligned.
image

**Comment by lafriks** [comment]
UI part looks odd. @silverwind maybe you have recommendations?

**Comment by lafriks** [comment]
Maybe it could be added between copy & download buttons after repo git link?

**Comment by 6543** [comment]
Add it as option to download menue?!?

**Comment by silverwind** [comment]
I feel we have too many buttons there already, so moving it to the download menu might be good to save space, but that menu might need to be renamed, so it has:

- Open in VSCode
- Download ZIP
- Download TAR.GZ

**Review by 6543 — APPROVED** [review]

**Review by KN4CK3R — APPROVED** [review]

**Pull request merged**